Tests/Unit/Extend/EmailTest.php

```diff
@@ -8,20 +8,55 @@
  *
  */
 
+use OxidEsales\Eshop\Application\Model\Article;
+use OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Application\Model\UserPayment;
+use OxidEsales\Eshop\Core\Field;
+
+use PHPUnit\Framework\MockObject\MockObject;
+
 use Wirecard\Oxid\Extend\Core\Email;
+use Wirecard\Oxid\Extend\Model\Basket;
 
-class EmailTest  extends OxidEsales\TestingLibrary\UnitTestCase
+class EmailTest extends OxidEsales\TestingLibrary\UnitTestCase
 {
+
+    /**
+     * @var Email|MockObject
+     */
+    private $_oEmail;
+
+    protected function setUp()
+    {
+        $this->_oEmail = oxNew(Email::class);
+
+        parent::setUp();
+    }
+
+    protected function failOnLoggedExceptions()
+    {
+        // don't fail on logged exception -> mailer is not instantiated
+        $this->exceptionLogHelper->clearExceptionLogFile();
+    }
+
     /**
      * @dataProvider testSendSupportEmailProvider
      */
     public function testSendSupportEmail($aEmailData)
     {
-        $oSmartyMock = $this->getMock("Smarty", array("fetch", "assign"));
-        $oSmartyMock->expects($this->any())->method("fetch")->will($this->returnValue(true));
+        $oSmartyMock = $this->getMockBuilder(Smarty::class)
```

Comment:
First part is the setup function. Then write multiple tests with one assert per test.